modules/component/schematics-core/tsconfig.lib.json

```diff
@@ -5,7 +5,7 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "downlevelIteration": true,
-    "outDir": "../../dist/modules/schematics-score",
```

Comment:
I actually like the nam schematics-core 👀

Reply:
Schematics do score, don't they 😄